src/environments/array.js

```diff
@@ -72,6 +72,9 @@ function parseArray(
             break;
         } else if (next === "\\\\" || next === "\\cr") {
             const cr = parser.parseFunction();
```

Comment:
In a followup PR, I'd like to have `parseFunction` throw if it was going return `null`.